gnosis/cowswap
# Update gnosis chain logo

# Summary

Fixes #2065 
![image](https://user-images.githubusercontent.com/622217/148826999-0546fedb-8bc0-4cb1-9ed6-7773701ac6bb.png)
![image](https://user-images.githubusercontent.com/622217/148827037-26c96198-595c-44d8-88d3-9dcfdafe1113.png)

  # To Test

1. Open the [Cowswap dApp](https://pr2073--gpswapui.review.gnosisdev.com/). And check the <img src="https://user-images.githubusercontent.com/622217/148829833-377e642a-cc83-4dbb-81e0-4f1ad6e9965e.png" height="25" /> Gnosis Chain logo in the network selector.
